docs/ReadmeExample.md

```diff
@@ -66,68 +58,73 @@ def fahrenheitToCelsius(f: Double): Double =
+// decoded: Stream[IO, String] = Stream(..)
+val lines: Stream[IO, String] = decoded.through(text.lines)
+// lines: Stream[IO, String] = Stream(..)
 ```
 
 Many of the functions defined for `List` are defined for `Stream` as well, for instance `filter` and `map`. Note that no side effects occur when we call `filter` or `map`. `Stream` is a purely functional value which can _describe_ a streaming computation that interacts with the outside world. Nothing will occur until we interpret this description, and `Stream` values are thread-safe and can be shared freely.
 
 ```scala
-scala> val filtered: Stream[IO, String] =
-     |   lines.filter(s => !s.trim.isEmpty && !s.startsWith("//"))
-filtered: fs2.Stream[cats.effect.IO,String] = Stream(..)
+val filtered: Stream[IO, String] =
+  lines.filter(s => !s.trim.isEmpty && !s.startsWith("//"))
+// filtered: Stream[IO, String] = Stream(..)
 
-scala> val mapped: Stream[IO, String] =
-     |   filtered.map(line => fahrenheitToCelsius(line.toDouble).toString)
-mapped: fs2.Stream[cats.effect.IO,String] = Stream(..)
+val mapped: Stream[IO, String] =
+  filtered.map(line => fahrenheitToCelsius(line.toDouble).toString)
+// mapped: Stream[IO, String] = Stream(..)
 ```
 
 Adds a newline between emitted strings of `mapped`.
 
 ```scala
-scala> val withNewlines: Stream[IO, String] = mapped.intersperse("\n")
-withNewlines: fs2.Stream[cats.effect.IO,String] = Stream(..)
+val withNewlines: Stream[IO, String] = mapped.intersperse("\n")
+// withNewlines: Stream[IO, String] = Stream(..)
 ```
 
 We use another pipe, `text.utf8Encode`, to convert the stream of strings back to a stream of bytes.
 
 ```scala
-scala> val encodedBytes: Stream[IO, Byte] = withNewlines.through(text.utf8Encode)
-encodedBytes: fs2.Stream[cats.effect.IO,Byte] = Stream(..)
+val encodedBytes: Stream[IO, Byte] = withNewlines.through(text.utf8Encode)
+// encodedBytes: Stream[IO, Byte] = Stream(..)
 ```
 
 We then write the encoded bytes to a file. Note that nothing has happened at this point -- we are just constructing a description of a computation that, when interpreted, will incrementally consume the stream, sending converted values to the specified file.
 
 ```scala
-scala> val written: Stream[IO, Unit] = encodedBytes.through(io.file.writeAll(Paths.get("testdata/celsius.txt"), blocker))
-written: fs2.Stream[cats.effect.IO,Unit] = Stream(..)
+val written: Stream[IO, Unit] = encodedBytes.through(io.file.writeAll(Paths.get("testdata/celsius.txt"), blocker))
+// written: Stream[IO, Unit] = Stream(..)
 ```
 
 There are a number of ways of interpreting the stream. In this case, we call `compile.drain`, which returns a val value of the effect type, `IO`. The output of the stream is ignored - we compile it solely for its effect.
 
 ```scala
-scala> val task: IO[Unit] = written.compile.drain
-task: cats.effect.IO[Unit] = <function1>
+val task: IO[Unit] = written.compile.drain
+// task: IO[Unit] = Map(
```

Comment:
FYI, there is a `mdoc:to-string` modifier to get the same formatted output as tut

Reply:
https://scalameta.org/mdoc/docs/modifiers.html#to-string

Reply:
Thanks!